frontend/benefit/handler/src/components/applicationReview/salaryBenefitCalculatorView/SalaryBenefitCalculatorView.tsx

```diff
@@ -372,10 +386,181 @@ const SalaryBenefitCalculatorView: React.FC<
           invalid={!!getErrorMessage(fields.endDate.name)}
           aria-invalid={!!getErrorMessage(fields.endDate.name)}
           errorText={getErrorMessage(fields.endDate.name)}
-          style={{ paddingRight: `${theme.spacing.xs}` }}
+          style={{ paddingRight: `${theme.spacing.s}` }}
         />
       </$GridCell>
 
+      {!isManualCalculator && (
+        <>
+          <$GridCell $colStart={1} $colSpan={11}>
+            <$CalculatorText
+              style={{ marginTop: theme.spacing.m, fontWeight: 'bold' }}
+            >
+              {t(`${translationsBase}.apprenticeshipCompensation`)}
+            </$CalculatorText>
+          </$GridCell>
+
+          {formik.values.trainingCompensations &&
```

Comment:
`formik.values.trainingCompensations?.length > 0 && ...`

Reply:
I will get error `Object is possibly 'undefined'.` (trainingCompensations?: TrainingCompensation[] | undefined) if used like this..